addons/web/static/src/views/fields/properties/property_definition.scss

```diff
@@ -62,3 +62,13 @@
         }
     }
 }
+
+.o_field_property_definition_type_popover {
```

Comment:
Can we also fix the dark mode while we're at it?

saas-17.1:
![image](https://github.com/odoo/odoo/assets/41282536/92370c4f-dbec-4801-bd2d-1704eadab0e6)

saas-17.2:
![image](https://github.com/odoo/odoo/assets/41282536/b19ed9ac-09ee-4173-8ca0-3ad84382a399)

I think in the past this was solved by Stéphane with:
https://github.com/odoo/enterprise/blob/master/web_enterprise/static/src/views/fields/properties/properties_field.dark.scss#L2

But it's probably not applied anymore?
To double-check (and clean the old rule if no longer useful).